app/code/Magento/Developer/Console/Command/QueryLogDisableCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\DeploymentConfig\Writer;
+use Magento\Framework\Config\File\ConfigFilePool;
+
+class QueryLogDisableCommand extends Command
+{
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:query-log:disable';
+
+    /**
+     * File logger alias
+     */
+    const QUIET_LOGGER_ALIAS = 'quiet';
+
+    /**
+     * @var Writer
+     */
+    private $deploymentConfigWriter;
+
+    public function __construct(
+        Writer $deploymentConfigWriter,
+        $name = null
+    )
+    {
+        parent::__construct($name);
+        $this->deploymentConfigWriter = $deploymentConfigWriter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName(self::COMMAND_NAME)
+            ->setDescription('Disable DB query logging');
+
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws \InvalidArgumentException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $data = ['db_logger_alias' => 'quiet'];
```

Comment:
- please use constants for db_logger_alias and quiet
- db_logger_alias is not the best name for the config parameter, because we used it to set the mode of the logger: "writing to file" vs "quiet". I would suggest something like "db logger mode"
- for the same reason, "quiet" might be renamed to "disabled"